sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -10,6 +10,30 @@ and includes six main functions:
 
 [Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_reference_doc] | [Product Documentation][product_documentation] | [Samples][samples_readme]
 
+## Table of contents
```

Comment:
curious why we need this? no other language has made this change and my understanding is that we want readme's to be as close as possible across languages

Reply:
we have this section in java other SDKs

Reply:
@czubair  or @kaerm for guidence here. Should we keep it? should other languages have this too?

Reply:
@czubair, @kaerm Curious do we have a final conclusion?

Reply:
from teams - @maggiepint said that there is no guidance here for now

Reply:
Thanks Kate. Shawn you ok if we delete this for now, and once the docs team has more information we can act accordingly?

Reply:
I can remove it for now.